src/supervisor-api.ts

```diff
@@ -87,6 +91,14 @@ export class SupervisorAPI {
 
 	private api = express();
 	private server: Server | null = null;
+	// Holds the function which should apply iptables rules
+	private applyRules: SupervisorAPI['applyListeningRules'] =
+		process.env.TEST === '1'
+			? () => {
+					// don't try to alter iptables
```

Comment:
Did prettier do this?

Reply:
I copied that code from a patch you sent me when we worked on another issue. I did not write it and prettier is ok with the format. What's the issue? 

The original code I copied is here: https://github.com/balena-io/balena-supervisor/pull/1279/commits/32379d925576a45616150d44d06dfb2030eb4b05#diff-67d01b95ff8ac8912ffcf19a8c9dcb20